docs/src/modules/components/MarkdownDocsV2.js

```diff
@@ -216,6 +217,42 @@ export default function MarkdownDocsV2(props) {
     return false;
   });
 
+  const scopedDemo = router.query.scopedDemo;
+  if (scopedDemo) {
+    return (
+      <AppFrame disableDrawer>
```

Comment:
Thank you, that was quick! 😅

Do we need the AppFrame for full-screen demos, though?
How about having a dedicated empty page, e.g. https://mui.com/playground?demo=material/components/buttons/BasicButtons.js
And a button in the toolbar that opens the playground url in a new tab?

Reply:
That would be harder to achieve but not impossible. in this PR we can basically surf on the existing generated infra for markdown demos.